rag/sql_rag/qdrant_vector_store.py

```
@@ -0,0 +1,155 @@
+      'Referer': 'https://68ab3360.us2a.app.preset.io'
+    }
+    response = requests.request("POST", url, headers=headers, data=json.dumps(payload))
+    print("run_sql:",response.text[:100])
+    if response.status_code != 200:
+        print(f"Error running SQL: {response.status_code} - {response.text}")
+        raise Exception(f"Error running SQL: {response.status_code} - {response.text}")
+    return pd.DataFrame(response.json()['data'])
+
+
+class IOIntelligence(VannaBase):
+  def __init__(self, config={}):
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)